librz/core/canalysis.c

```diff
@@ -398,13 +414,28 @@ static void bb_info_print(RzCore *core, RzAnalysisFunction *fcn, RzAnalysisBlock
 			rz_cons_printf(" f 0x%08" PFMT64x, bb->fail);
 		}
 		if (bb->switch_op) {
-			RzAnalysisCaseOp *cop;
-			RzListIter *iter;
-			RzList *unique_cases = rz_list_uniq(bb->switch_op->cases, casecmp);
-			rz_list_foreach (unique_cases, iter, cop) {
-				rz_cons_printf(" s 0x%08" PFMT64x, cop->addr);
+			RzAnalysisCaseOp *case_op;
+			void **it;
+			RzPVector *unique_cases = rz_pvector_new(NULL);
+			rz_pvector_foreach (bb->switch_op->cases, it) {
+				case_op = *it;
+				bool found = false;
+				void **it_unique;
+				rz_pvector_foreach (unique_cases, it_unique) {
+					if (casecmp(*it_unique, case_op) == 0) {
+						found = true;
+						break;
+					}
+				}
```

Comment:
how about using rz_pvector_find ?

Reply:
Sure, I will use it instead of `rz_pvector_foreach`, will surely simply it.

Reply:
Sure, using `rz_pvector_find` will optimize it. Will use that instead.